File: 1000-digit-fibonacci-number/1000-digit-fibonacci-number.py
sequence = [1,1]

def fib(n):
	if n == 1 or n == 2:
		return sequence[n-1]
	sequence.append(sequence[n-3] + sequence[n-2])
	return sequence[n-1]

i = 1
found = False
while not found:
	term = str(fib(i))
	if len(term) == 1000:
		found = True
		print i
	i += 1

File: 1000-digit-fibonacci-number/1000-digit-fibonacci-number.js
var sequence = [1,1];
function fib (n) {
    if (n == 1 || n == 2) {
        return sequence[n-1];       
    }
    else {
        sequence = sequence.concat(sequence[n-3] + sequence[n-2]);
        return sequence[n-1];
    }
};
function fibOneThousand(output = 'csv') {
    console.log('Starting the count to one thousand.');
    var i = 1;
    var n = 1000;
    while ( i <= n ) {
        var out = fib(i);
        if (output == 'verbose') {
            console.log("Fibonacci number " , i , " is ", out);
        }
        else if(output == 'csv') {
            console.log(i, out);
        }
        i++;
    }
    console.log("The final Fibonacci number is ", out);
    return;
}
fibOneThousand();